conf/rpb-titel-to-strapi.fix

```diff
@@ -34,7 +34,8 @@ do put_macro("add_url")
   if exists("$[from]")
     if all_match("$[from]", "(https?|ftp):\\/\\/(\\S+\\.)+\\S+(\\/\\S*)?|")
       move_field("$[from]", "f90[].$append.f90_")
-      add_field("f90[].*.n90", "$[type]")
+      #add_field("f90[].*.n90", "$[type]")
```

Comment:
I think this commented line could/should go:

```suggestion

```

Looks good otherwise. Will deploy data to test and update UI to show the new fields.